h/views.py

```diff
@@ -7,10 +7,13 @@
 from pyramid.events import ContextFound
 from pyramid.view import forbidden_view_config, notfound_view_config
 from pyramid.view import view_config
+import pyramid.response
 
 from . import session
 from .models import Annotation
 from .resources import Application, Stream
+import h.api_client.api_client as api_client
+import h.atom_feed
```

Comment:
These imports should be a bit cleaner now (it's just `import h.api_client`) but I don't like `from foo import bar`. The Python docs [warn against it](https://docs.python.org/2/howto/doanddont.html#from-module-import-name1-name2). One problem it causes is that if I want to patch `h.api_client.Client` in a test (lets say it's a high-level function test that runs code from many h modules), then I have to find every different instance of `from .api_client import Client` and patch each one seperately. For example if `h/views.py` does `from .api_client import Client` I now have to patch `h.views.Client` and so on.

Just doing `import .api_client` instead of `import h.api_client` might be fine but I'm not sure so I'm inclined not to save one character.